miniapp/USERGUIDE.md

```diff
@@ -206,52 +291,138 @@ val userInfoBridgeDispatcher = object : UserInfoBridgeDispatcher() {
 ```
 
-`MiniAppDisplay.navigateBackward` and `MiniAppDisplay.navigateForward` facilitates the navigation inside a mini app if the history stack is available in it. A common usage pattern could be to link it up to the Android Back Key navigation.
+### Screen Orientation
+**API Docs:** [MiniAppMessageBridge.allowScreenOrientation](api/com.rakuten.tech.mobile.miniapp.js/-mini-app-message-bridge/)
 
-**Note:** Clearing up the mini app display is essential. `MiniAppDisplay.destroyView` is required to be called when exit miniapp.
+The default setting does not allow miniapp to change hostapp screen orientation.
+Hostapp can allow miniapp to control the screen orientation for better experience by calling 
 
-**Note:** There are several different types of exceptions which could be thrown by `MiniApp.create`, but all are sub-classes of `MiniAppSdkException`.
-You can handle each exception type differently if you would like different behavior for different cases.
-For example you may wish to display a different error message when the server contains no published versions of a mini app.
-See the full list of exceptions in the [API docs](api/com.rakuten.tech.mobile.miniapp/-mini-app/create.html).
+```kotlin
+miniAppMessageBridge.allowScreenOrientation(true)
+```
 
-**Note:** Preview Mode
+In case miniapp is allowed to control, please ensure that your activity handles screen orientation.
+There are several ways to prevent the view from being reset.
+In our Demo App, we set the config on activity `android:configChanges="orientation|screenSize"`.
+See [here](https://developer.android.com/guide/topics/resources/runtime-changes#HandlingTheChange).
+
+## Fetching Mini App Info
+**API Docs:** [MiniApp.listMiniApp](api/com.rakuten.tech.mobile.miniapp/-mini-app/list-mini-app.html), [MiniApp.fetchInfo](api/com.rakuten.tech.mobile.miniapp/-mini-app/fetch-info.html), [MiniAppInfo](api/com.rakuten.tech.mobile.miniapp/-mini-app-info/)
+
+Information about Mini Apps can be fetched in two different ways: by using `MiniApp.listMiniApp` to get a list of info for all Mini Apps, or by using `MiniApp.fetchInfo` to get info for a single Mini App. Either method will return `MiniAppInfo` objects with info about the Mini App such as name, icon URL, ID, version, etc.
+
+Use `MiniApp.listMiniApp` if you want a list of all Mini Apps:
+
+```kotlin
+CoroutineScope(Dispatchers.IO).launch {
+    try {
+        val miniAppList = MiniApp.instance().listMiniApp()
+    } catch(e: MiniAppSdkException) {
+        Log.e("MiniApp", "There was an error retrieving the list", e)
+    }
+}
+```
+
+Or use `MiniApp.fetchInfo` if you want info for a single Mini App and already know the Mini App's ID:
+
+```kotlin
+CoroutineScope(Dispatchers.IO).launch {
+    try {
+        val miniAppInfo = MiniApp.instance().fetchInfo("MINI_APP_ID")
+    } catch(e: MiniAppSdkException) {
+        Log.e("MiniApp", "There was an error retrieving the Mini App info", e)
+    }
+}
+```
+
+**Note:** Preview Mode: 
```

Comment:
I think the note of preview mode should be put under #3 Create and display a Mini App

Reply:
@minh-rakuten Thanks, I've updated this. I also noticed that there were a lot of notes under that section so I re-formatted it into a list of notes.